src/test/java/fr/insalyon/pldagile/tsp/SimulatedAnnealingTest.java

```diff
@@ -64,7 +66,7 @@ void runSimulatedAnnealing() {
     }
 
     @Test
-    void getTotalDistance() {
+    void getTotalDistance() throws InterruptedException {
```

Comment:
We should test if when the exception is thrown it invalidates the test or if we should catch it instead

Reply:
there's actually no InterruptedException